.github/workflows/continuous_testing.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  schedule:
+    - cron: '30 18 * * *'
+
+jobs:
+  test_live:
+    name: Station Recording Validation
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
+    env:
+      DISCORD_ACCESS_TOKEN: ${{ secrets.DISCORD_ACCESS_TOKEN }}
```

Comment:
Is this used?

Reply:
Hmm, I found the answer: Yes it is used to set access token for OneSDK when running against live API.

But now I wonder if it is possible to use different slack instance for testing and for reporting.

Reply:
I think it is a good idea, even though both tokens will be the same now.